Resources/Private/Extensions/News/Templates/News/Detail.html

```diff
@@ -189,40 +189,55 @@ <h2>
 				</n:extensionLoaded>
 			</f:if>
 
-			<f:if condition="{settings.backPid}">
-				<!-- Link Back -->
-				<div class="news-backlink-wrap">
-					<f:link.page pageUid="{settings.backPid}">
-						<f:translate key="back-link" />
-					</f:link.page>
-				</div>
-			</f:if>
-
 			<f:if condition="{settings.detail.showPrevNext}">
 				<f:comment>Show next and previous news item</f:comment>
 				<n:simplePrevNext pidList="{newsItem.pid}" news="{newsItem}" as="paginated" sortField="datetime">
 					<div class="page-navigation">
 						<f:if condition="{paginated}">
-							<ul class="prev-next">
-								<f:if condition="{paginated.prev}">
-									<li class="previous">
-										<n:link newsItem="{paginated.prev}" settings="{settings}">
-											<f:translate key="paginate_previous"/>
+							<f:if condition="{paginated.prev}">
+								<div class="page-navigation__prev">
+									<div class="page-navigation__link-wrp js__dotdotdot">
+										<n:link newsItem="{paginated.prev}" settings="{settings}" class="page-navigation__link">
+											{paginated.prev.title}
 										</n:link>
-									</li>
-								</f:if>
-								<f:if condition="{paginated.next}">
-									<li class="next">
-										<n:link newsItem="{paginated.next}" settings="{settings}">
-											<f:translate key="paginate_next"/>
+									</div>
+									<span class="page-navigation__date">
+											<time datetime="{f:format.date(date:paginated.prev.datetime, format:'Y-m-d')}">
+											<f:format.date format="{f:translate(key:'dateFormat')}">{paginated.prev.datetime}</f:format.date>
+											<meta itemprop="datePublished" content="{f:format.date(date:paginated.prev.datetime, format:'Y-m-d')}"/>
+											</time>
+										</span>
+								</div>
+							</f:if>
+							<f:if condition="{paginated.next}">
+								<div class="page-navigation__next">
+									<div class="page-navigation__link-wrp js__dotdotdot">
+										<n:link newsItem="{paginated.next}" settings="{settings}" class="page-navigation__link">
+											{paginated.next.title}
 										</n:link>
-									</li>
-								</f:if>
-							</ul>
+									</div>
+									<span class="page-navigation__date">
+											<time datetime="{f:format.date(date:paginated.next.datetime, format:'Y-m-d')}">
+											<f:format.date format="{f:translate(key:'dateFormat')}">{paginated.next.datetime}</f:format.date>
+											<meta itemprop="datePublished" content="{f:format.date(date:paginated.next.datetime, format:'Y-m-d')}"/>
+											</time>
+										</span>
+								</div>
+							</f:if>
 						</f:if>
 					</div>
 				</n:simplePrevNext>
 			</f:if>
+
+			<f:if condition="{settings.backPid}">
+				<!-- Link Back -->
+				<div class="news-backlink-wrap">
+					<f:link.page pageUid="{settings.backPid}">
+						<f:translate key="back-link" />
+					</f:link.page>
+				</div>
+			</f:if>
+		</f:then>
```

Comment:
Looks like this tag `</f:then>` is not needed - cause errors.

Reply:
Also, could you explain how to enable navigation for next and prev news in detail view template? I would like to test it, but can't find this feature :)

Reply:
Do you mean "_plugin.tx_news.settings.detail.showPrevNext = 1_" in EXT:theme_t3kit/Resources/Private/Extensions/News/TypoScript/setup.ts? It should be enabled.

Reply:
Yes, I still can't enable those links, looks like I am missing some config ... :(
But I decided to merge it without testing, hope that everything should be ok. Thanks for the help!